src/EnergyPlus/PackagedThermalStorageCoil.cc

```diff
@@ -2727,62 +2751,68 @@ void SizeTESCoil(EnergyPlusData &state, int &TESCoilNum)
+                state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).FluidStorageVolume =
+                    (state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).DischargeOnlyRatedDischargeCap *
+                     state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).StorageCapacitySizingFactor * DataGlobalConstants::SecInHour) /
+                    (rho * Cp * deltaT);
+            } else {
+                state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).FluidStorageVolume =
+                    (state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).CoolingOnlyRatedTotCap *
+                     state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).StorageCapacitySizingFactor * DataGlobalConstants::SecInHour) /
+                    (rho * Cp * deltaT);
+            }
+            BaseSizer::reportSizerOutput(state,
+                                         "Coil:Cooling:DX:SingleSpeed:ThermalStorage",
+                                         state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).Name,
+                                         "Fluid Storage Volume [m3]",
+                                         state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).FluidStorageVolume);
         }
-        BaseSizer::reportSizerOutput(state,
-                                     "Coil:Cooling:DX:SingleSpeed:ThermalStorage",
-                                     state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).Name,
-                                     "Fluid Storage Volume [m3]",
-                                     state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).FluidStorageVolume);
-    }
-    if ((state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).StorageMedia == MediaType::IceBased) &&
-        (state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).IceStorageCapacity == DataGlobalConstants::AutoCalculate)) {
-
-        if (state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).DischargeOnlyRatedDischargeCap > 0.0 &&
-            state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).DischargeOnlyModeAvailable) {
-            state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).IceStorageCapacity =
-                state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).DischargeOnlyRatedDischargeCap *
-                state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).StorageCapacitySizingFactor * DataGlobalConstants::SecInHour;
-        } else {
-            state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).IceStorageCapacity =
-                state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).CoolingOnlyRatedTotCap *
-                state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).StorageCapacitySizingFactor * DataGlobalConstants::SecInHour;
+        break;
+    case MediaType::Ice:
+        if (state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).IceStorageCapacity == DataGlobalConstants::AutoCalculate) {
```

Comment:
70% of the text in this file is `state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum);`.